net/shorewall/Makefile

```diff
@@ -0,0 +1,118 @@
+#
+# Copyright (C) 2008-2012 OpenWrt.org
+# Copyright (C) 2017 Willem van den Akker <wvdakker@wilsoft.nl>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=shorewall
+PKG_VERSION:=5.1.4.1
+PKG_DIRECTORY:=5.1
+PKG_MAINVERSION:=5.1.4
+PKG_RELEASE:=2
+
+PKG_SOURCE_URL:=http://www.shorewall.net/pub/shorewall/$(PKG_DIRECTORY)/shorewall-$(PKG_MAINVERSION)/ 
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_HASH:=4650e2ec73dcf6c38cf2992fefdbd2e302a34a9687a933d8a375d227df2621db96c9a2e54ebbfac50014152caffc0736a7bc1192f479c69c56f78f7874bc09b3
```

Comment:
typo ? sha256sum c561dd91c4b88ef087a6d172bafb4ac5d01c0055c47b855568536a0c422ff232
on 6 the same

Reply:
Dont understand what you mean. Hashes are different.

Or did you mean PKG_HASH must be SHA256sum?

Reply:
on me is failing with no md5sum or sha256sum format type

Reply:
IT has to be PKG_HASH and we only accept sha256 hashes.